src/features/search/KennisartikelDetail.vue

```diff
@@ -29,9 +29,10 @@
   </article>
 
   <content-feedback
+    v-if="currentFeedbackSection"
```

Comment:
unrelated, in some circumstances the current section could be undefined. this was causing errors.